keyboards/hineybush/h87_g2/info.json

```
@@ -23,96 +31,554 @@
         "pid": "0x0001",
         "vid": "0x4069"
     },
+    "community_layouts": ["tkl_ansi", "tkl_ansi_split_bs_rshift", "tkl_ansi_tsangan", "tkl_ansi_tsangan_split_bs_rshift"],
+    "layout_aliases": {
+        "LAYOUT_all": "LAYOUT_tkl_ansi_split_bs_rshift"
```

Backwards compatibility for those using, what would be post-merge, previous layout name in their personal keymaps
```suggestion
        "LAYOUT_all": "LAYOUT_tkl_ansi_split_bs_rshift",
        "LAYOUT": "LAYOUT_tkl_ansi_tsangan"
```